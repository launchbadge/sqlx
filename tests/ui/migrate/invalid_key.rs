fn main() {
    //Fails due to invalid key
    sqlx::migrate!("foo", parameters = [(123, "foo")]);
}
