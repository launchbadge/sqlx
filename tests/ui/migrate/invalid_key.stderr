error: proc macro panicked
 --> tests/ui/migrate/invalid_key.rs:3:5
  |
3 |     sqlx::migrate!("foo", parameters = [(123, "foo")]);
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = help: message: Expected a string literal or a variable in key
  = note: this error originates in the macro `sqlx::migrate` (in Nightly builds, run with -Z macro-backtrace for more info)
